specs/deneb/polynomial-commitments.md

```diff
@@ -414,6 +387,51 @@ def verify_kzg_proof_impl(commitment: KZGCommitment,
     ])
 ```
 
+#### `verify_kzg_proof_multi`
+
+```python
+def verify_kzg_proof_multi(commitments: Sequence[KZGCommitment],
+                           zs: Sequence[BLSFieldElement],
+                           ys: Sequence[BLSFieldElement],
+                           proofs: Sequence[KZGProof]) -> bool:
```

Comment:
I believe these parameter types should be `Bytes48` and `Bytes32` for consistency. Especially since `bls.bytes48_to_G1(commitment)` is used below. You'd want to rename the parameter names too, to append "_bytes" to each one.

Reply:
What's everyone's opinion on removing the "_bytes" from parameter names?

Reply:
I don't think so? It's the same as `verify_kzg_proof_impl` and only used internally, so it seems right to me. But @asn-d6 can comment maybe?

Reply:
Oh it's only used internally, disregard then.